smarts/env/utils/observation_conversion.py

```diff
@@ -707,6 +714,9 @@ def name(self):
             "speed": gym.spaces.Box(
                 low=0, high=1e10, shape=(_NEIGHBOR_SHP,), dtype=np.float32
             ),
+            "interest": gym.spaces.Box(
```

Comment:
Since `interest` is a binary space it should be either `gymnasium.spaces.Discrete(n=2)` or `gymnasium.spaces.MultiBinary(n=1)` .